exercises/pascals-triangle/canonical-data.json

```diff
@@ -80,7 +80,7 @@
           "input": {
             "count": -1
           },
-          "expected": -1
+          "expected": {"error": "count must be positive"}
```

Comment:
Out of scope for this PR, but (opinion) this test should be removed.

Reply:
@Insti your meaning that, in your opinion, this case should be removed because it deals with invalid input?  I agree with that opinion.  If I am understanding you correctly I will leave this as it is.  Another PR can be opened to to remove this case.

Reply:
@rpottsoh you understand me and I agree with you. :smile: